doc/development/source/orange-demo/orangedemo/OWDataSamplerA.py

```diff
@@ -3,16 +3,20 @@
 import numpy
 
 import Orange.data
-from Orange.widgets import widget, gui
+from Orange.widgets.widget import OWWidget, Input, Output
+from Orange.widgets import gui
 
-class OWDataSamplerA(widget.OWWidget):
+class OWDataSamplerA(OWWidget):
     name = "Data Sampler"
     description = "Randomly selects a subset of instances from the data set"
     icon = "icons/DataSamplerA.svg"
     priority = 10
 
-    inputs = [("Data", Orange.data.Table, "set_data")]
-    outputs = [("Sampled Data", Orange.data.Table)]
+    class Inputs:
+        data = Input("Data", Orange.data.Table)
+
+    class Outputs:
+        sample = Output("Sampled Data", Orange.data.Table)
```

Comment:
[Zen of python, line 2](https://www.python.org/dev/peps/pep-0020/#id3).

The way the code is written now, someone might even be able to figure out what is going on. If we us the "magic" way, PyCharm and PyLint would complain, which usually means that it is not that obvious what is going on ("you say in the class definition that Outputs.data is Table, but now you call Outputs.data.send()??? Table has no send????)